app/services/idv/session.rb

```diff
@@ -1,8 +1,6 @@
 module Idv
   class Session
     VALID_SESSION_ATTRIBUTES = %i[
-      async_result_id
-      async_result_started_at
```

Comment:
😂 
As Andy would say, "time is a flat circle"
https://github.com/18F/identity-idp/pull/1524/files#diff-2df0b9c9e83aa13bc56217ce31c8040c